src/PureDataNode.cpp

```diff
@@ -142,6 +147,10 @@ void PureDataNode::sendMessage(const std::string &dest, const std::string &msg,
   queueTask([=](pd::PdBase &pd) { pd.sendMessage(dest, msg, list); });
 }
 
+void PureDataNode::sendNoteOn(int port, int pitch, int velocity) {
+  queueTask([=](pd::PdBase &pd) { pd.sendNoteOn(port, pitch, velocity); } );
```

Comment:
Since you've made a new `QueueItem` type, it should be used here. See how it's done in `sendFloat` etc.